internal/automation-scripts/DownloadProductsZip.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PRODUCT_HOME="$HOME/products"
+PRODUCT_ZIP_HOME="$HOME/productZips"
+SCRIPT_TAG="[SEC_AUTOMATION_UPDATE_PRODUCTS]"
+URL_PREFIX="https://api.github.com/repos/wso2/"
+URL_SUFIX="releases?client_id="
+
+echo "$SCRIPT_TAG [START]"
+
+echo "$SCRIPT_TAG Cleaning previous Product Downloads"
+rm -rf $PRODUCT_ZIP_HOME
+
+echo "$SCRIPT_TAG Cleaning product home: ($PRODUCT_HOME)"
+rm -rf $PRODUCT_HOME
+mkdir $PRODUCT_HOME
+
+for url in $(cat $HOME/scripts/config/WeeklyReleasedSupportedProductList.conf)
+do
+    mkdir $PRODUCT_ZIP_HOME
+    echo "$URL_PREFIX$url/$URL_SUFIX$(cat $HOME/scripts/config/GitHubAppToken.conf)"
```

Comment:
Add $SCRIPT_TAG, which is used to track what script is adding the log line (concurrent situation). Also log what is going to happen with this path.